.github/workflows/changelog.yml

```diff
@@ -0,0 +1,94 @@
+name: changelog
+
+on:
+  pull_request:
+    types: [opened, synchronize, labeled, unlabeled, reopened]
+
+permissions:
+  contents: read
+
+env:
+  # Also change CACHE_VERSION in the other workflows
+  CACHE_VERSION: 8
+  DEFAULT_PYTHON: 3.8
+
+jobs:
+  prepare-base:
```

Comment:
Let's merge this with the other job, that saves time 😄

Reply:
Done 👍